adm_program/languages/de-DE.xml

```diff
@@ -1364,6 +1364,25 @@
   <string name="SYS_TEST_MAIL">Testmail</string>
   <string name="SYS_TEST_MAIL_DESC">Bevor Sie diese Funktion aufrufen, müssen alle Einstellungen für den E-Mail-Versand gespeichert werden!\nEs wird eine einfache E-Mail mit dem Betreff #VAR1_BOLD# an die E-Mail-Adresse der angemeldeten Benutzerin bzw. des angemeldeten Benutzers auf Grundlage der gespeicherten E-Mail-Einstellungen versendet. Dabei wird beim SMTP-Versand ein Versandprotokoll ausgegeben, welches bei Problemen helfen kann.</string>
   <string name="SYS_TEXT">Text</string>
+  <string name="SYS_TFA">Zwei-Faktor-Authentifizierung</string>
+  <string name="SYS_TFA_ENABLE">Zwei-Faktor-Authentifizierung (2FA) aktivieren</string>
+  <string name="SYS_TFA_ENABLE_DESC">Aktivieren Sie die Zwei-Faktor-Authentifizierung (2FA) für alle Benutzer, um einen zusätzlichen Verifizierungsschritt zu erfordern, der die Kontosicherheit erhöht und vor unbefugtem Zugriff schützt.</string>
+  <string name="SYS_TFA_ENTER">Bitte geben Sie den Sicherheits Code aus Ihrer OTP App ein.</string>
+  <string name="SYS_TFA_LOST">Ich habe keinen Zugang zu meinem 2. Faktor</string>
+  <string name="SYS_TFA_PROTECTED_CONTENT">Dieser Inhalt ist durch Zwei Faktor Authentifizierung (2FA) geschützt.</string>
+  <string name="SYS_TFA_RESET">Zwei-Faktor-Authentifizierung zurücksetzen</string>
+  <string name="SYS_TFA_RESET_DESC">Entfernen Sie den konfigurierten zweiten Faktor.</string>
+  <string name="SYS_TFA_RESET_SUCCESSFUL">Zwei-Faktor-Authentifizierung erfolgreich zurückgesetzt.</string>
+  <string name="SYS_TFA_SETUP">Zwei-Faktor-Authentifizierung einrichten</string>
+  <string name="SYS_TFA_SETUP_INSTRUCTIONS">Um die Zwei-Faktor-Authentifizierung (2FA) einzurichten, befolgen Sie bitte die folgenden Schritte:</string>
+  <string name="SYS_TFA_SETUP_INSTRUCTIONS_STEP_1">Scannen Sie den QR-Code: Verwenden Sie eine Authentifizierungs-App (z.B. Google Authenticator, Authy), um den angezeigten QR-Code zu scannen.</string>
+  <string name="SYS_TFA_SETUP_INSTRUCTIONS_STEP_2">Geben Sie den Sicherheitscode ein: Öffnen Sie die Authentifizierungs-App, suchen Sie den für unseren Dienst generierten Authentifizierungscode und geben Sie ihn in das Eingabefeld ein.</string>
+  <string name="SYS_TFA_SETUP_INSTRUCTIONS_STEP_3">Bestätigen: Senden Sie den Sicherheitscode, um die Einrichtung abzuschließen.</string>
+  <string name="SYS_TFA_SETUP_SCAN_QR">QR-Code</string>
```

Comment:
This Text-Key should only be named SYS_QR_CODE , so it's easier for others to reuse that string.